### PR TITLE
V1.3.1 update userguide

### PR DESCRIPTION
**Pull request by marcustw**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -349,54 +377,139 @@ Examples:
...
+```
 
-The user will be prompted to confirm their decision here.
+**Expected result:**
+```
+Delete INDEX? (yes/no)
+```
 
-If yes: deletes the specified student from the student list.
+:tip: After receiving the confirmation prompt, enter `yes` to confirm the removal of student at the specified `INDEX`, or else, reject the command by entering `no`.
```
Comment (CodingCookieRookie) [inline docs/UserGuide.md]: Perhaps could be better phrased to ":tip: After receiving the confirmation prompt, enter `yes` to confirm the removal of student at the specified `INDEX` or reject the command by entering `no`"?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -349,54 +377,139 @@ Examples:
...
 
-The user will be prompted to confirm their decision here.
+Clears and removes all students from the student list.
+
+<div markdown="block" class="alert alert-info">
 
-If yes: removes all students from the student list.
+**:information_source: Notes about the usage of this command:**<br>
+* The user will be prompted to confirm their decision here.
+   * If yes: removes all students from the student list.
```
Comment (CodingCookieRookie) [inline docs/UserGuide.md:587]: Perhaps could have more consistent usage of active vs passive voice?

File: docs/UserGuide.md [hunk 3]
```diff
@@ -518,18 +631,47 @@ Examples:
 
 [todo]
 
+{ start of `features#userconfirmation#clearstu` written by: Marcus Tan Wei }
+
 ### 5.7. User confirmation
 
-Prompts the user to confirm the execution of commands that may permanently remove information.
+Some commands have the ability to alter the data stored in the hard disk.
+ATAS aims to make careful decisions and further prompts the user to confirm the execution of those commands.
```
Comment (CodingCookieRookie) [inline docs/UserGuide.md]: Perhaps ATAS should be bold for consistency?

**Review by CodingCookieRookie — APPROVED** [review]
LGTM with minor nitpicks

File: docs/UserGuide.md (hunk 2 above)
Comment (nweiyue) [reply docs/UserGuide.md:587]: Use of "you" here?

**Review by nweiyue — APPROVED** [review]
LGTM!

**Pull request merged**